osm-processor/start.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+echo $OMS_FILE_ACTION
```

Comment:
@Rub21 do you mean to call this `OMS_FILE_ACTION` or should it be `OSM_FILE_ACTION` ?

Reply:
Thank you @batpad for noticing it !! I have fixed the issue